README.md

```diff
@@ -112,8 +112,8 @@ bundle exec rake ssg:import_rhel_supported
 The compliance project integrates with project cyndi. For local development, a database view is created, built from the inventory database which runs alongside the compliance database. The Cyndi hosts view exists within an inventory schema within the compliance database.
 
 ```shell
-bundle exec rails db < db/cyndi_setup.sql # syndicated (cyndi) hosts from inventory
-RAILS_ENV=test bundle exec rails db < db/cyndi_setup.sql # cyndi for test DB
+bundle exec rails db < db/cyndi_setup_devel.sql # syndicated (cyndi) hosts from inventory
+RAILS_ENV=test bundle exec rails db < db/cyndi_setup_test.sql # cyndi for test DB
```

Comment:
@vkrizan check out this solution to testing. In devel, the view comes from the inventory table. In test environments (local or CI), it comes from a local table like the Cyndi onboarding specifies. This seems to work great. I still have some work to go through the tests and remove any places that try to add/update hosts, but I'm excited about this working solution. :)

Reply:
:+1: Thanks!